### PR TITLE
[8.0] base_import_odoo: optionally define extra fields to read for use in postprocessing

### PR DESCRIPTION
**Pull request by thomaspaulb**

### CODE REVIEW

**Comment by thomaspaulb** [comment]
@hbrunn Reading all fields had some other complications, so I made `extra_fields` optional and to be defined by the user. It seems a generic enough addition to warrant a PR.

File: base_import_odoo/models/import_odoo_database.py [hunk 1]
```diff
@@ -184,15 +184,19 @@ def _run_import_model(self, context):
         """Import records of a configured model"""
         model = self.env[context.model_line.model_id.model]
         fields = self._run_import_model_get_fields(context)
+        local_fields = fields.keys()
+        extra_fields = []
+        if context.model_line.extra_fields:
+            extra_fields = safe_eval(context.model_line.extra_fields)
```
Comment (hbrunn) [inline base_import_odoo/models/import_odoo_database.py:190]: yay, that was exactly what I disliked about the first version

Comment (hbrunn) [reply base_import_odoo/models/import_odoo_database.py:190]: ... maybe I merged to enthousiastically. Why `safe_eval` and not `.split(',')` + `filtered`?

Comment (thomaspaulb) [reply base_import_odoo/models/import_odoo_database.py:190]: I mimiced what you did some lines below for the "defaults" dictionary.

Comment (hbrunn) [reply base_import_odoo/models/import_odoo_database.py:190]: yes, but the defaults must be a dict. Now I need to write valid python code and not just field1,field2 etc. But leave as is, this is not worth raising noise I think after reconsidering

**Pull request merged**